### ISSUE TITLE
IE11: centered layout param not centering vertically

### ISSUE THREAD

**Issue by petermikitsh** [storybookjs/storybook#12386]
**Describe the bug**

Stories center horizontally, but not vertically, in IE 11.

**To Reproduce**

1. Enable the centered addon, and visit the story in IE 11.

**Expected behavior**

Vertical centering occurs.

**Screenshots**

The official storybook is not loading in IE 11.
<img width="1280" alt="Screen Shot 2020-09-04 at 10 15 55 AM" src="https://user-images.githubusercontent.com/1571918/92269446-0b95e000-ee99-11ea-9892-fdd5f805c7d0.png">

My personal storybook is, showing the vertical entering bug:
<img width="1280" alt="Screen Shot 2020-09-04 at 10 34 00 AM" src="https://user-images.githubusercontent.com/1571918/92270059-392f5900-ee9a-11ea-99f4-f64b035265d4.png">

Expected behavior (centering in Chrome is correct):
<img width="1392" alt="Screen Shot 2020-09-04 at 10 35 59 AM" src="https://user-images.githubusercontent.com/1571918/92270226-857a9900-ee9a-11ea-8511-79df3d6949d1.png">

**Code snippets**

```tsx
// preview.tsx

export const parameters = {
  layout: 'centered',
};
```

**System:**
Please paste the results of `npx sb@next info` here.

```
Environment Info:

  System:
    OS: macOS Mojave 10.14.6
    CPU: (12) x64 Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
  Binaries:
    Node: 12.18.0 - ~/.nvm/versions/node/v12.18.0/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 6.14.4 - ~/.nvm/versions/node/v12.18.0/bin/npm
  Browsers:
    Safari: 12.1.2
  npmPackages:
    @storybook/addon-actions: ^6.0.5 => 6.0.16 
    @storybook/addon-essentials: ^6.0.5 => 6.0.16 
    @storybook/addon-links: ^6.0.5 => 6.0.16 
    @storybook/addon-storyshots: ^6.0.12 => 6.0.16 
    @storybook/addon-storyshots-puppeteer: ^6.0.12 => 6.0.16 
    @storybook/addon-storysource: ^6.0.5 => 6.0.16 
    @storybook/addons: ^6.0.5 => 6.0.16 
    @storybook/react: ^6.0.5 => 6.0.16 
```

**Comment by ndelangen** [comment]
@petermikitsh would you be able to contribute a fix?

**Comment by ghengeveld** [comment]
This is a known bug in IE11 and there appears to be a workaround outlined in the highest rated comment here: https://github.com/philipwalton/flexbugs/issues/231

Some ideas:

Add a pseudo element after the flex element (in our case the body).
```css
body:after {
  content: '';
  min-height: inherit;
  font-size: 0;
}
```

Someone also suggested setting `flex-direction: column;` on the flex element. No idea whether that would help. I also found `flex-basis: 100%` somewhere but it might interfere when the component is taller than the viewport height.

**Comment by shilman** [comment]
Huzzah!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.1.0-rc.5 containing PR #13159 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb upgrade --prerelease
```

Closing this issue. Please re-open if you think there's still more to do.

**Comment by stefanprobst** [comment]
this change in #13159 actually added an unnecessary vertical scrollbar when using a `centered` layout in Firefox and Chrome:

```css
  /* Vertical centering fix for IE11 */
  .sb-show-main.sb-main-centered:after {
    content: '';
    min-height: inherit;
    font-size: 0;
  }
```

when changing `min-height` to `height` it's fine, but i cannot test what this does in IE11 

do you want me to open a new issue for this?

**Comment by petermikitsh** [comment]
@stefanmirck The scrollbar bug was captured in https://github.com/storybookjs/storybook/issues/13178#issuecomment-731274291 and https://github.com/storybookjs/storybook/pull/13192

**Comment by stefanprobst** [comment]
i guess this can be closed again after #13192 was merged? thanks!
